### PR TITLE
docs: add docker compose docs

### PR DESCRIPTION
**Pull request by mojojoji**
This PR adds documentation on how to use the native `docker compose` integration of CodeSandbox.

Content has been added to the following pages
- https://wwgpm4-3000.csb.app/learn/environment/docker#configuring-docker-compose
- https://wwgpm4-3000.csb.app/tutorial/getting-started-with-docker#4-create-docker-compose-file

Fixes PTI-230 https://linear.app/codesandbox/issue/PTI-230/document-docker-compose-integration-in-public-docs

### CODE REVIEW

**Review by acekyd — APPROVED** [review]

File: packages/projects-docs/pages/tutorial/getting-started-with-docker.mdx [hunk 1]
```diff
@@ -47,3 +47,23 @@ Now that the container has been built, all your new terminal sessions will be ex
 Note that you also have `root` access inside the terminal, that's because we give root access by default inside a container. With this, you can install new packages and use them in your sandbox from the terminal as well. Keep in mind that it's still better to configure this in the `Dockerfile` in the long term, as these temporary container changes won't persist between Dockerfile rebuilds.
 
 ![htop running inside the container](/docker-terminal.png)
+
+### 4. Create docker compose file
+Once the environment is setup with `.codesandbox/Dockerfile`, you can run additional services that are required for the dev environment using the native docker compose integration in CodeSandbox.
```
Comment (filipelima18) [inline packages/projects-docs/pages/tutorial/getting-started-with-docker.mdx:52]:
```suggestion
Once the environment is set up with `.codesandbox/Dockerfile`, you can run additional services that are required for the dev environment using the native docker compose integration in CodeSandbox.
```

File: packages/projects-docs/pages/tutorial/getting-started-with-docker.mdx [hunk 2]
```diff
@@ -47,3 +47,23 @@ Now that the container has been built, all your new terminal sessions will be ex
 Note that you also have `root` access inside the terminal, that's because we give root access by default inside a container. With this, you can install new packages and use them in your sandbox from the terminal as well. Keep in mind that it's still better to configure this in the `Dockerfile` in the long term, as these temporary container changes won't persist between Dockerfile rebuilds.
 
 ![htop running inside the container](/docker-terminal.png)
+
+### 4. Create docker compose file
```
Comment (filipelima18) [inline packages/projects-docs/pages/tutorial/getting-started-with-docker.mdx:51]:
```suggestion
### 4. Create Docker Compose file
```

File: packages/projects-docs/pages/tutorial/getting-started-with-docker.mdx [hunk 3]
```diff
@@ -47,3 +47,23 @@ Now that the container has been built, all your new terminal sessions will be ex
...
+Once the environment is setup with `.codesandbox/Dockerfile`, you can run additional services that are required for the dev environment using the native docker compose integration in CodeSandbox.
+
+Create a new file `docker-compose.yml` in `.codesandbox` folder. As an example you can add the following contents in the file to run an nginx server.
```
Comment (filipelima18) [inline packages/projects-docs/pages/tutorial/getting-started-with-docker.mdx:54]:
```suggestion
Create a new file `docker-compose.yml` in the `.codesandbox` folder. For example, you can add the following contents in the file to run an nginx server.
```

**Review by filipelima18 — COMMENTED** [review]
Added some suggestions @mojojoji

**Comment by mojojoji** [comment]
@filipelima18 applied all the suggested changes

**Review by filipelima18 — APPROVED** [review]

**Pull request merged**